model/src/main/kotlin/PackageCurationResult.kt

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2017-2018 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package com.here.ort.model
+
+/**
+ * This class contains information about which values were changed when applying a curation.
+ */
+data class PackageCurationResult(
+        /**
+         * Contains the values from before applying the [curation]. Values which were not changed are null.
+         */
+        val before: PackageCurationData,
```

Comment:
Nit: If this is called `before`, I would have expected the below to be called `after` :-) But as below you're not storing the final data, but the curation that is applied (i.e. the "diff"), how about calling this "original" or "base" instead?

Reply:
Done.